src/BizHawk.BizInvoke/BizHawk.BizInvoke.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFramework>net48</TargetFramework>
+		<TargetFrameworks>net48;net6.0</TargetFrameworks>
```

Comment:
This shouldn't be necessary?

Reply:
There are issues with certain hacks in the code and `#if NET_6_0` defines that can sometimes cause missing functions on runtime when not every project has a separate .net framework and .net 6 target, and so to make sure everything works I've multi-targeted every single project instead of just the output projects.

Reply:
This might be because I've been mistakenly using `NET6_0` instead of `NET6_0_OR_GREATER`...